### ISSUE TITLE
The next to last column in the version list is somewhat messy.

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7397]
In the same column we have an icon linking to documentation if that exists, nothing if no documentation is generated or another icon if analysis failed for that version.

It is not completely clear what we want to show in that column, perhaps we need to rethink it or have more explanation.

**Comment by sigurdm** [comment]
![image](https://github.com/dart-lang/pub-dev/assets/8613953/f5c2c904-30b2-413e-a8b2-1bd40b877e5c)

**Comment by isoos** [comment]
The column is an overview of what documentation (for which version) we have for the package. Not sure if users also recognize the intent there though.

**Comment by sigurdm** [comment]
> Not sure if users also recognize the intent there though.

Yeah - I think that's the point. We could give the column a header to explain. What would that header say?

**Comment by isoos** [comment]
> Yeah - I think that's the point. We could give the column a header to explain. What would that header say?

If we put text there, the column gets much wider (same with the downloads). I remember an initial design that had text in there, but got dropped in favor of the compactness.
